src/components/Main.ui.js

```diff
@@ -463,18 +463,6 @@ export class MainComponent extends React.Component<Props> {
             />
           </Stack>
 
-          <Stack key={Constants.PLUGIN_EARN_INTEREST}>
```

Comment:
I'll admit I don't know anything about this plugin.
Are you sure we don't use/need it anywhere?

Reply:
Cred took our customer's money and ran, so Paul made a public statement that we will never integrate a custodial service again. If customers want interest going forward, they can buy some DeFi interest-bearing tokens. We can promote these tokens in the app using our normal promo-card stuff.